### DISCUSSION TITLE
QOL - List Receiver Methods from highlighted scripts

### DISCUSSION THREAD

**Discussion by joshthemisanthrope** [godotengine/godot-proposals#4526] [Editor]
It would be nice to have a drop down of all the methods in a selected script when connecting a signal through the editor. It could just be a little dropdown arrow on the right of the Receiver Method text bar.

It would definitely improve workflow and ease of use for signals through the editor. Right now I have to go into my script to check the function name to make sure I have it verbatim.

Thoughts? I might even take a stab at it when I'm in the mood. Doesn't seem like a very hard feature to implement.

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/239. There's a [pull request](https://github.com/godotengine/godot/pull/38218) implementing this, but there are disagreements as to the UX it provides.
